common/app/views/fragments/items/elements/facia_cards/image.scala.html

```diff
@@ -27,6 +28,6 @@
     }
     <!--[if IE 9]></video><![endif]-->
     @maybeImageMedia.map(ImgSrc.getFallbackUrl(_)).orElse(maybeSrc).orElse(maybePath.map(ImgSrc(_, ImageProfile(width = widths.desktop.map(_.get))))).map { src =>
-        <img loading="@{ if (Switches.LazyLoadImages.isSwitchedOn) "lazy" else "auto"}" class="@RenderClasses(classes: _*)" alt="" src="@src"/>
+        <img loading="@{ if (Switches.LazyLoadImages.isSwitchedOn && lazyLoadIndex) "lazy" else "auto"}" class="@RenderClasses(classes: _*)" alt="" src="@src" id="@lazyLoadIndex"/>
```

Comment:
I don't think we want to add `lazyLoadIndex` as an `id` here as from what I understand, `lazyLoadIndex` is a container index, so will not be unique per image?